boxoffice/views/admin.py

```diff
@@ -42,6 +45,26 @@ def org(organization):
     return dict(org=organization, title=organization.title)
 
 
+@app.route('/admin/o/<org>/ic/new', methods=['POST'])
+@lastuser.requires_login
+@xhr_only
+@load_models(
+    (Organization, {'name': 'org'}, 'organization'),
+    permission='org_admin'
+    )
+def admin_new_ic(organization):
+    ic_form = ItemCollectionForm()
+    if ic_form.validate_on_submit():
+        ic = ItemCollection(organization=organization)
+        ic_form.populate_obj(ic)
+        if not ic.name:
+            ic.make_name()
+        db.session.add(ic)
+        db.session.commit()
+        return api_success(result={'item_collection': dict(ic.access_for(user=g.user))}, doc=_(u"New item collection created"), status_code=201)
+    return api_error(message=_(u"Incorrect data"), errors=ic_form.errors, status_code=400)
```

Comment:
`Incorrect data` is a little vague. Better to return the form's validation messages?